### PR TITLE
Refactor to allow download of generated report

### PR DESCRIPTION
**Pull request by crashGoBoom**
# Description

Hi @statik !

So here is what I have so far with downloading the generated reports. This uses the RPC to grab the file for download.

### CODE REVIEW

**Comment by crashGoBoom** [comment]
@statik Hi! This will now all the user to download the file from the db using the file name that it was inserted into the db with.

File: webui/Caddyfile-dev [hunk 1]
```diff
@@ -35,6 +35,11 @@ proxy /api api:8180 {
   transparent
 }
 
+proxy /rpc api:8180 {
+  transparent
```
Comment (statik) [inline webui/Caddyfile-dev]: @crashGoBoom does this need a corresponding change in Caddyfile to work in production?

File: flyway/sql/V28__no_anonymous_on_registration_funnel_1.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-- remove support for anonymous insert on registration_funnel_1
+REVOKE ALL ON mappa.registration_funnel_1 FROM anonymous;
+GRANT INSERT ON mappa.registration_funnel_1 to member;
```
Comment (statik) [inline flyway/sql/V28__no_anonymous_on_registration_funnel_1.sql]: still don't quite understand this, but i'm happy to leave something that works. i imagine we'll need to take a more comprehensive review of the permissions and make sure everything is lining up in the future.

File: webui/Caddyfile-dev (hunk 1 above)
Comment (crashGoBoom) [reply webui/Caddyfile-dev]: @statik yes! good call i'll work on that

**Pull request merged**